test/test.options.bool.js

```diff
@@ -1,15 +1,28 @@
-/**
- * Module dependencies.
- */
+const commander = require('../');
+// eslint-disable-next-line no-unused-vars
+const should = require('should');
```

Comment:
You might just want to do `require('should');` (unless that violates another ESLint rule...)

Reply:
Oh nice suggestion, matches intentions, I'll try that.

Reply:
Done